.github/workflows/flutter_ci.yml

```diff
@@ -19,9 +19,9 @@ jobs:
         # flutter-version: '1.12.x' # you can also specify exact version of flutter
     
     - run: flutter pub get    
-    - run: flutter format --set-exit-if-changed .    
+    #- run: flutter format --set-exit-if-changed .    
     - run: flutter analyze .    
-    - run: flutter test    
+    #- run: flutter test    
```

Comment:
@patwalnik Why these 2 are commented?

Reply:
1. flutter format is used to format code base. not used rn
2. codebase has no test case written so it will throw error.

Reply:
Alright. So I think since it is boilerplate, let's un-comment the format code one command, keeping test script commented.
@patwalnik

Reply:
Sure,
But I would suggest you to run that command locally first.
I have used it personally in my project and sometimes it throws error if things are not properly done.

Reply:
Yeah. that should be done locally of course.